controller/src/main/java/io/pravega/controller/store/stream/Stream.java

```diff
@@ -233,11 +234,11 @@
     /**
      * Heartbeat method to keep transaction open for at least lease amount of time.
      *
-     * @param txId  Transaction identifier.
+     * @param txData Transaction data.
      * @param lease Lease period in ms.
      * @return Transaction metadata along with its version.
      */
-    CompletableFuture<VersionedTransactionData> pingTransaction(final UUID txId, final long lease);
+    CompletableFuture<VersionedTransactionData> pingTransaction(final VersionedTransactionData txData, final long lease);
```

Comment:
To be consistent with other parts of the code and since we are changing this parameter, let's call it `txnData`.